src/socket.c

```diff
@@ -719,6 +728,30 @@ gboolean socket_lock_input_cb(GIOChannel *source, GIOCondition condition, gpoint
 				cl_options.goto_column = atoi(buf);
 			}
 		}
+		else if (strncmp(buf, "desktop_startup_id", 18) == 0)
+		{
+			if (socket_fd_gets(sock, buf, sizeof(buf)) != -1 && *buf != '.')
+			{
+#if defined(GDK_WINDOWING_WAYLAND) || defined(GDK_WINDOWING_X11)
+				g_strstrip(buf); /* remove \n char */
+				geany_debug("got desktop_startup_id: %s", buf);
```

Comment:
Maybe also, could the string be modified to something similar to https://github.com/geany/geany/blob/22a8a1f55e30db8566f752211bcd895c707ea24c/src/socket.c#L678 so it's a little more descriptive? E.g. something like

```
geany_debug("Received IPC command from remote instance: desktop_startup_id (%s)", buf)
```